### DISCUSSION TITLE
Using Pinia alongside Vuex

### DISCUSSION THREAD

**Discussion by Dawid-Ohia** [vuejs/pinia#2534] [Help and Questions] [answered]
Hi,

 _"It is recommended to avoid using both Pinia and Vuex"_
from  https://pinia.vuejs.org/ssr/nuxt.html#Using-Pinia-alongside-Vuex

Does this recommendation apply to Pinia and Vuex in general or just to Nuxt?

**Comment by posva** [comment, marked as answer]
It has more implications in Nuxt that in plain Vue apps because of the auto imports and other automatic behaviors that Nuxt add. You might find yourself with implicit things happening and that's always difficult to bedug
